app/src/test/java/ch/hikemate/app/authentication/AuthViewModelTest.kt

```diff
@@ -143,7 +143,7 @@ class AuthViewModelTest {
     // Verify that currentUser is initially null
     assertNull(viewModel.currentUser.first())
 
-    viewModel.signInWithEmailAndPassword("mock@example.com", "password")
+    viewModel.signInWithEmailAndPassword("mock@example.com", "password", {}, {})
```

Comment:
Nitpick: you might want to make a test where you check that `onSuccess` or `onFailure` are called properly.

Reply:
Your test will succeed if neither `onSuccess` nor `onFailure` is called. Is that OK?

Reply:
This shouldn't happen. I think we're good with that

Reply:
Could happen that we forget it when we modify the view model

Reply:
My bad, I did not see the check for the user afterwards. Forget I said anything...